code/.ipynb_checkpoints/COVID19 US county Map-Daily Update-checkpoint.ipynb

```diff
@@ -43,414 +48,26 @@
     "tdst=df_Counties_confirmed.columns[-1]\n",
     "print (tdst)\n",
     "tdst1=df_Counties_deaths.columns[-1]\n",
-    "print (tdst1)"
+    "print (tdst1)\n",
+    "\n",
+    "#Change date on yesterday's csv and zipped shapefile in TEST folder\n",
```

Comment:
This is a really good idea to do the date shifting and file name changes in the script, rather than manually -- if that's what I'm seeing here.  I'm not sure why it's in the checkpoint and not in the main commit, however.

Reply:
That might be a case of me not using github properly, or I may not have saved the file before I submitted it - should be resolved once I export to a .py file.

Reply:
Sweet - my understanding is that it the notebook creates a checkpoint on each save.  Possible you also committed the checkpoint but not the changes on the main scripts?  Anyway - also see my note about removing the checkpoints from git entirely.  I'd be happy to push a commit to do this.